05/rc_homework.py

```diff
@@ -0,0 +1,66 @@
+def get_num(rc):
+    return int(''.join(x for x in rc if x.isdigit()))
+
+
+def check_format(rc):
+    if '/' not in rc or len(rc) != 11:
+        return Exception('Wrong format. The length must be 11.')
```

Comment:
Toto je špatné použitý vyjímky. V toto případě to vrátí objekt `Exception`, ale vyjímku nevyhodí. 
Při testu `if Exception('Lalala'): print('OK')` se vyhodnotí jako True. Takže i v tvém případě se kontrola na 11 znaků a lomítko provede ale je vždycky True. 
Co jsi pravděpodobně chtěla je vyhození, tedy `raise Excetiption('Lalala')`.

Nicméně bych to v tomto případě nedoporučil, protože pro ostatní kontroly používáš `return True/False` + `print()`. Držel bych se tedy stejného stylu.